backend/attendance/src/main/resources/attendance-service-persister.yml

```
@@ -0,0 +1,408 @@
+
+            - jsonPath: $.staff.*.denrollmentDate
+
+            - jsonPath: $.staff.*.additionalDetails
+              type: JSON
+              dbType: JSONB
+
+            - jsonPath: $.staff.*.auditDetails.lastModifiedBy
+
+            - jsonPath: $.staff.*.auditDetails.lastModifiedTime
+
+            - jsonPath: $.staff.*.id
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)